docs/sources/panels-visualizations/configure-tooltips/index.md

```diff
@@ -0,0 +1,108 @@
+---
+aliases:
+keywords:
+  - grafana
+  - tooltips
+  - documentation
+labels:
+  products:
+    - cloud
+    - enterprise
+    - oss
+menuTitle: Configure tooltips
+title: Configure tooltips
+description: Configure tooltips for your visualizations
+weight: 75
+---
+
+# Configure tooltips
+
+When you hover your cursor over a visualization, Grafana can display tooltips that contain more information about a data point like the exact time of a result. You can customize tooltips to to control how many series they include and the order of those values. You can also copy the content from tooltips to use elsewhere. Learn more about configuring tooltips in [Tooltip options](#tooltip-options).
```

Comment:
maybe `...data point, like the exact time of a result.`? and an extra "to" here -`You can customize tooltips to to control`